hazelcast/src/main/java/com/hazelcast/instance/DefaultNodeContext.java

```diff
@@ -150,26 +150,27 @@ private EventLoopGroup createEventLoopGroup(Node node, NodeIOService ioService)
         MemberChannelInitializer initializer
                 = new MemberChannelInitializer(loggingService.getLogger(MemberChannelInitializer.class), ioService);
 
-        ChannelErrorHandler exceptionHandler
+        ChannelErrorHandler errorHandler
                 = new TcpIpConnectionChannelErrorHandler(loggingService.getLogger(TcpIpConnectionChannelErrorHandler.class));
 
         if (spinning) {
             return new SpinningEventLoopGroup(
                     loggingService,
                     node.nodeEngine.getMetricsRegistry(),
-                    exceptionHandler,
+                    errorHandler,
                     initializer,
                     node.hazelcastInstance.getName());
         } else {
             return new NioEventLoopGroup(
-                    loggingService,
-                    node.nodeEngine.getMetricsRegistry(),
-                    node.hazelcastInstance.getName(),
-                    exceptionHandler,
-                    ioService.getInputSelectorThreadCount(),
-                    ioService.getOutputSelectorThreadCount(),
-                    ioService.getBalancerIntervalSeconds(),
-                    initializer);
+                    new NioEventLoopGroup.Context()
+                            .loggingService(loggingService)
+                            .metricsRegistry(node.nodeEngine.getMetricsRegistry())
+                            .threadNamePrefix(node.hazelcastInstance.getName())
+                            .errorHandler(errorHandler)
+                            .inputThreadCount(ioService.getInputSelectorThreadCount())
+                            .outputThreadCount(ioService.getOutputSelectorThreadCount())
+                            .balanceIntervalSeconds(ioService.getBalancerIntervalSeconds())
```

Comment:
maybe named as `balancerIntervalSeconds` or `ioBalancerIntervalSeconds`

Reply:
good spotting. The inconsistency isn't nice.